src/core/drive/navigator.ts

```diff
@@ -96,6 +96,7 @@ export class Navigator {
     if (responseHTML) {
       const snapshot = PageSnapshot.fromHTMLString(responseHTML)
       await this.view.renderPage(snapshot)
+      window.scroll(0, 0)
```

Comment:
@sstephenson this used to be passed as a callback, but then the rendering interface was changed. Is this still the correct spot for it?